src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -151,6 +151,12 @@ export function PanelHeader({
       </span>
     );
   };
+  
+  if (hidePanelTitle) {
+   return (
+  <div data-test-subj="dashboardPanelTitle__wrapper" className={classes} style={{visibility:'collapse'}}></div> 
```

Comment:
Hey there! By returning early here with only the `dashboardPanelTitle__wrapper` element, you have made it so users can no longer edit their panels - i.e., on hover, the panel header no longer shows up. 

The goal here should be to **only** render the `wrapper` element when `hidePanelTitle` is false, but you should still render the panel bar in either case. So while your previous solution worked better than this one, my recommendation is to do what you did before but without the same code duplication 👍